packages/suite/src/components/suite/Preloader/Preloader.tsx

```diff
@@ -31,6 +33,11 @@ const ROUTES_TO_SKIP_REVISION_CHECK: RouterAppWithParams['app'][] = [
     'firmware-custom',
 ];
 
+const LeftWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+`;
+
```

Comment:
Could you please use `<Row>...</Row>` from `@trezor/components` instead of this? It allows us to have less duplicit components